### DISCUSSION TITLE
IS this still actively/heavily developed?

### DISCUSSION THREAD

**Discussion by J-HS-L** [sqlitebrowser/sqlitebrowser#3287] [Q&A] [answered]
It doesn't seem to have had a major release in quite some time.

Thank you!

**Comment by justinclift** [comment, marked as answer]
It's very much still actively developed, and yeah... we're very overdue for getting another release out the door.

In the meantime, if you're wanting to have access to our latest features then the nightly builds are pretty good (also pretty stable). :smile:

**Comment by chrisjlocke** [comment]
The commit history would also suggest development is still ongoing.
https://github.com/sqlitebrowser/sqlitebrowser/commits/master
